app/assets/javascripts/delete.js.es6

```diff
@@ -1,17 +1,17 @@
-'use strict'
+'use strict';
 
 $(() => {
     deleteIdea();
-})
+});
 
-var deleteIdea = () => {
+let deleteIdea = () => {
     $('#ideas').delegate('#delete-idea', 'click', (event) => {
         let $idea = $(event.toElement);
         $.ajax({
             type: 'DELETE',
             url: '/api/v1/ideas/' + $idea.attr('idea-id'),
-            success: () => { $idea.parent().remove() },
-            error: (xhr) => { alert(xhr.responseTest) }
-        })
-    })
-}
+            success: () => { $idea.parent().remove(); },
+            error: (xhr) => { alert(xhr.responseTest); }
```

Comment:
I still feel like either the callback or the promise-style pattern is a better fit here.